modules/yieldmoBidAdapter.js

```diff
@@ -40,9 +40,12 @@ export const spec = {
       w: localWindow.innerWidth,
       h: localWindow.innerHeight
     };
-    for (var request of bidRequests) {
+    // for (var request in bidRequests) {
```

Comment:
should probably just remove these comments?  definitely shouldn't change them to signaling `for...in` loops were being used as that could confuse others down the line since they behave differently than the `for...of` loops that were there originally.

Reply:
was playing around with both loops. Removed dead code now.